src/styles/_catalog.scss

```diff
@@ -14,6 +18,11 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition-duration: 300ms;
+    &:hover {
+      color: #34568b;
+      transform: translateZ(100px);
+    }
```

Comment:
you should specify transition properties directly
default all, but it could affect old browsers' performance